lib/msf/core/db_manager/http/http_db_manager_service.rb

```diff
@@ -16,6 +16,15 @@ def start(opts)
 
     require_environment!(parsed_options)
 
+    if opts[:ssl]
+      ssl_opts = {}
+      ssl_opts[:private_key_file] = opts[:ssl_key]
+      ssl_opts[:cert_chain_file] = opts[:ssl_cert]
+      ssl_opts[:verify_peer] = false
```

Comment:
Should `:verify_peer` be set `true`?

Reply:
I think this goes with the above comment. Will adjust functionality based on what we figure out there.

Reply:
I just tested it with the server set to `true` and no changes on the client and it still works. I'm not  sure that this option actually works?